elements/pfe-card/demo/index.html

```diff
@@ -186,7 +187,7 @@ <h2>Imaged card with side overflow</h2>
         </pfe-cta>
       </pfe-card>
 
-      <pfe-card pfe-color="lighter">
+      <pfe-card pfe-color="darker" pfe-img-src="https://www.redhat.com/files/webux/img/bandbg/bg-infrastructure-management-pattern-transparent-tile.png" style="--pfe-card__header--BackgroundColor: rgba(45,45,45,.7);">
```

Comment:
Should we use a local fixture instead of a dotcom image?

Reply:
ah yes, thx!

Reply:
I split this card into two cards, one w/ the image on the bottom, the other w/ the background img. And changed the redhat.com img to a placeholder.